xcube_stac/store.py

```diff
@@ -0,0 +1,410 @@
+        return self.stac.open_data(data_id, **open_params)
+
+    def describe_data(
+        self, data_id: str, **open_params
+    ) -> DatasetDescriptor:
+        """ Get the descriptor for the data resource given by *data_id*.
+
+        Args:
+            data_id (str): An identifier of data that is provided by this
+                store.
+
+        Raises:
+            NotImplementedError: Not implemented yet.
+
+        Returns:
+            DatasetDescriptor: data descriptor containing meta data of
+                the data resources identified by *data_id*
+        """
+        # ToDo: implement describe_data method.
+        raise NotImplementedError("describe_data() operation is not supported yet")
+
+
+class StacDataStore(StacDataOpener, DataStore):
+    """ STAC implementation of the data store.
+    """
+
+    def __init__(self, **stac_kwargs):
+        """
+        Args:
+            **stac_kwargs: Parameters required by the STAC data store.
+                * url (str): URL to STAC catalog (required)
+                * collection_prefix (str): Path of collection used as
+                    entry point (optional)
+                * data_id_delimiter (str): Delimiter used to separate
+                    collections, items and assets from each other (optional)
+        """
+        super().__init__(stac=Stac(**stac_kwargs))
+
+    @classmethod
+    def get_data_store_params_schema(cls) -> JsonObjectSchema:
+        """ Get the JSON schema for instantiating a new data store.
+
+        Returns:
+            JsonObjectSchema: The JSON schema for the data store's parameters.
+        """
+        stac_params = dict(
+            url=JsonStringSchema(
+                title="URL to STAC catalog"
+            ),
+            collection_prefix=JsonStringSchema(
+                title="Collection prefix",
+                description="Path of collection used as entry point",
+            ),
+            data_id_delimiter=JsonStringSchema(
+                title="Data ID delimiter",
+                description=(
+                    "Delimiter used to separate collections, "
+                    "items and assets from each other"
+                ),
+            )
+        )
+        return JsonObjectSchema(
+            properties=stac_params,
+            required=["url"],
+            additional_properties=False
+        )
+
+    @classmethod
+    def get_data_types(cls) -> Tuple[str, ...]:
+        """ Get alias names for all data types supported by this store.
+
+        Returns:
+            Tuple[str, ...]: The tuple of supported data types.
+        """
+        return (DATASET_TYPE.alias,)
+
+    def get_data_types_for_data(self, data_id: str) -> Tuple[str, ...]:
+        """ Get alias names for of data types that are supported
+        by this store for the given *data_id*.
+
+        Args:
+            data_id (str): An identifier of data that is provided by this
+                store.
```

Comment:
```suggestion
        Args:
            data_id: An identifier of data that is provided by this store.
```

No need to repeat the type here.